src/overlays/actors/ovl_En_Bom/z_en_bom.c

```diff
@@ -279,7 +279,8 @@ void EnBom_Update(Actor* thisx, GlobalContext* globalCtx) {
                 effPos.y += 30.0f;
             }
 
-            func_80028E84(globalCtx, &effPos, &effVelocity, &bomb2Accel, 0x64, (thisx->shape.rot.z * 6) + 0x13);
+            EffectSsBomb2_SpawnLayered(globalCtx, &effPos, &effVelocity, &bomb2Accel, 100,
+                                         (thisx->shape.rot.z * 6) + 19);
```

Comment:
I'm wondering if these numbers (or at least the addition) should be in hex since this is a rotation value

Reply:
ah, I see now it looks like it was a conscious choice. maybe I'm out of the loop

Reply:
z rotation is often used as an extension of params. in this case the arg is how many units the scale of the effect should increment every frame. z rotation in bombs seems to be related to the explosion radius, for some reason

but yea its not angle related in any way

Reply:
Makes sense, thanks for explaining.